artifacts/definitions/Windows/Network/NetstatEnriched.yaml

```diff
@@ -0,0 +1,168 @@
+    type: hidden
+    default: |
+      Choice,Regex
+      ALL,".*"
+      ESTABLISHED,"^ESTAB$"
+      LISTENING,"^LISTEN$"
+      OTHER,"CLOS|SENT|RCVD|LAST|WAIT|DELETE"
+
+  - name: ProcessNameRegex
+    description: "regex search over source process name"
+    default: ".*"
+  - name: ProcessPathRegex
+    description: "regex search over source process path"
+    default: ".*"
+  - name: CommandLineRegex
+    description: "regex search over source process commandline"
+    default: ".*"
+  - name: HashRegex
+    description: "regex search over source process hash"
+    default: ".*"
+  - name: UsernameRegex
+    description: "regex search over source process user context"
+    default: ".*"
+  - name: AuthenticodeSubjectRegex
+    description: "regex search over source Authenticode Subject"
+    default: ".*"
+  - name: AuthenticodeIssuerRegex
+    description: "regex search over source Authenticode Issuer"
+    default: ".*"
+  - name: AuthenticodeVerified
+    description: "Authenticode signiture selection"
+    type: choices
+    default: ALL
+    choices:
+       - ALL
+       - TRUSTED
+       - UNSIGNED
+       - NOT TRUSTED
+  - name: AuthenticodeVerifiedMap
+    type: hidden
+    default: |
+      Choice,Regex
+      ALL,".*"
+      TRUSTED,"^trusted$"
+      UNSIGNED,"^unsigned$"
+      NOT TRUSTED,"unsigned|disallowed|untrusted|error"
+
+sources:
+  - name: Netstat
+    queries:
+      - LET VerifiedRegex <= SELECT Regex 
+            FROM parse_csv(filename=AuthenticodeVerifiedMap, accessor="data")
+            WHERE Choice=AuthenticodeVerified LIMIT 1
+      - LET StatusRegex <= SELECT Regex 
+            FROM parse_csv(filename=StatusMap, accessor="data")
+            WHERE Choice=Status LIMIT 1
+      - LET FamilyRegex <= SELECT Regex 
+            FROM parse_csv(filename=FamilyMap, accessor="data")
+            WHERE Choice=Family LIMIT 1
+      - LET TypeRegex <= SELECT Regex 
+            FROM parse_csv(filename=TypeMap, accessor="data")
+            WHERE Choice=Type LIMIT 1
+            
+      - LET process <= SELECT Pid as PsId,
+            Ppid,
+            Name, 
+            CommandLine, 
+            Exe, 
+            Hash,
+            Authenticode,
+            Username
+        FROM Artifact.Windows.System.Pslist()
+        WHERE Name =~ ProcessNameRegex 
+
+      - SELECT Pid,
+            { SELECT Ppid FROM process WHERE PsId = Pid } as Ppid,
+            { SELECT Name FROM process WHERE PsId = Pid } as Name,
+            { SELECT Exe FROM process WHERE PsId = Pid } as Path,
+            { SELECT CommandLine FROM process WHERE PsId = Pid } as CommandLine,
+            { SELECT Hash FROM process WHERE PsId = Pid } as Hash,
+            { SELECT Username FROM process WHERE PsId = Pid } as Username,
+            { SELECT Authenticode FROM process WHERE PsId = Pid } as Authenticode,
+            FamilyString as Family,
+            TypeString as Type,
+            Status,
+            LaddrIP, LaddrPort,
```

Comment:
Change here the same `Laddr.IP AS LaddrIP`

Reply:
Modded accross all relevant files:
Laddr.IP AS LaddrIP, Laddr.Port AS LaddrPort,
Raddr.IP AS RaddrIP, Raddr.Port AS RaddrPort,